src/core.c

```diff
@@ -2208,30 +2208,52 @@ void SaveStorageValue(int position, int value)
 #endif
 
     int dataSize = 0;
+    int new_dataSize = 0;
```

Comment:
Hey, I avoid snake_case variables in raylib, please use camelCame (`newDataSize`).

Reply:
By the way: https://github.com/raysan5/raylib/wiki/raylib-coding-conventions